src/wx/cmdevents.cpp

```diff
@@ -2766,10 +2769,26 @@ EVT_HANDLER_MASK(ChangeFilter, "Change Pixel Filter", CMDEN_NREC_ANY)
     }
 
     wxString msg;
-    msg.Printf(_("Using pixel filter %s"), pixel_filters->GetString(gopts.filter));
+    msg.Printf(_("Using pixel filter:") + " %s", pixel_filters->GetString(gopts.filter));
```

Comment:
Not every language will put the value at the end of the sentence, so the formatter should be part of the string to localize.